google/cloud/storage/tests/BUILD.bazel

```diff
@@ -54,3 +54,31 @@ VARIATIONS = {
         "@com_google_googletest//:gtest_main",
     ],
 ) for test in storage_client_integration_tests for v_label, v_env in VARIATIONS.items()]
+
+[cc_test(
+    name = test.replace("/", "_").replace(".cc", ""),
```

Comment:
QQ: can you explain why we're doing this?

Reply:
My understanding is that this is creating the binary name by replacing the extension / slashes with the appropriate replacements. Looks like this is pretty consistent across our bazel files.